runtime/onert/backend/cl_common/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+file(GLOB SOURCES "*.cc")
```

Comment:
'cl-common' seems header-only library. If so, 'SOURCES' looks redundant this line and the next lines including seems at this momenthis moment. Am I missing something?

Reply:
I'll fix it.